docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3"
+services:
+  web:
+    # replace username/repo:tag with your name and image details
+    image: metabrainnetwork
+    build: ./
+    ports:
+      - "1337:1337"
+      - "4000:85"
+    depends_on:
+      - redis
+      - elasticsearch
+      - predictionserver
+      - pwasserver
+    volumes:
+        - "/data:/data"
+    environment:
+      - "discovery.zen.ping.unicast.hosts=elasticsearch"
+      - ELASTICSEARCH_URL=http://elasticsearch:9200"
+      - ELASTICSEARCH_HOST=elasticsearch
+  redis:
+    image: redis
+    ports:
+      - "6379:6379"
+  elasticsearch:
+    image: elasticsearch:2.4
+    ports:
```

Comment:
Use expose instead of the port binding. The expose variant will only make the port available within the docker-compose stack itself.